imports/client/components/ChatPeople.tsx

```diff
@@ -118,6 +124,8 @@ class ChatPeople extends React.Component<ChatPeopleProps, ChatPeopleState> {
       error: '',
       muted: false,
       deafened: false,
+      callersExpanded: true,
+      viewersExpanded: false,
```

Comment:
Can I convince you to start with viewers expanded as well?

I think it's easier to collapse things if you discover that you want to see more of the chat backscroll than it is to notice the viewer count and ambiently become aware of other people that might be motivated to join a call if you were the first to click the "join audio call" button.